src/foam/nanos/analytics/LogVisitAgent.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * @license
+ * Copyright 2024 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.analytics',
+  name: 'LogVisitAgent',
+  documentation: `
+    log an analyticevent when certain urlsearchparams are present
+  `,
+
+  methods: [
+    function init() {
+      this.logVisit();
+    },
+    function logVisit() {
```

Comment:
Please make this a listener instead of a method is put isMerged: true, mergeDelay: 20000, so that it doesn't run immediately. We like to delay this kind of non time critical work so that it doesn't hinder our startup speed.